repos/system_upgrade/el7toel8/actors/multipathconfcheck/libraries/library.py

```diff
@@ -0,0 +1,130 @@
+from leapp.reporting import create_report
+from leapp import reporting
+
+
+def _merge_configs(configs):
+    options = {"default_path_checker": None, "detect_prio": None,
+               "detect_path_checker": None, "reassign_maps": None,
+               "retain_attached_hw_handler": None}
+    for config in configs:
```

Comment:
The way multipath's configuration works, it processes the main file /etc/multipath.conf first, and then all the files in the config dir in alphabetical order, and each file can override options in files that came before. If you look at _parse_config_dir in the MultipathConfRead library, it sorts the files, and then reads them, so that they are added to the configs list in order.